packages/js-client/package.json

```
@@ -0,0 +1,76 @@
+{
+  "name": "@aragon/sdk-client-template",
+  "author": "Aragon Association",
+  "version": "1.0.0",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=16"
+  },
+  "scripts": {
+    "start": "tsdx watch",
+    "build": "tsdx build",
+    "test": "TESTING=true tsdx test --verbose --runInBand",
+    "lint": "tsdx lint",
+    "prepare": "tsdx build",
```

```suggestion
    "prepare": "yarn build",
```
Why the duplication when the two are identical?

---

This is generated by tsdx. Its true that is exactly the samee thing so i'll delete it